runtime/runtime-core/common/src/aws/smithy/kotlin/runtime/util/Concurrency.kt

```diff
@@ -0,0 +1,11 @@
+package aws.smithy.kotlin.runtime.util
+
+import aws.smithy.kotlin.runtime.InternalApi
+import kotlinx.coroutines.runBlocking
+
+@InternalApi
+public fun runBlocking(block: suspend () -> Unit) {
+    runBlocking {
+        block()
+    }
+}
```

Comment:
question: What is this used for?

Reply:
It's so we can have `runBlocking` as a runtime type, it's used in the presigner generator

Reply:
You can just add a new `runBlocking` type under the existing `KotlinxCoroutines` runtime type